DOCS-820: Refactor Install Guide IA pass (#1365)
viamrobotics/docs: docs/installation/manage/_index.md

#### Start

```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
sudo systemctl start viam-server
```

#### Stop

```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
sudo systemctl stop viam-server
```

#### Restart

```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
sudo systemctl restart viam-server
```

#### Enable (start automatically with system boot, default)

```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
sudo systemctl enable viam-server
```

#### Disable (do not start automatically with system boot)

```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
sudo systemctl disable viam-server
```

<br>

### Run from the command line

When running `viam-server` on the command line, you can use the following commands to manage the process.
If `viam-server` is already running as a system service, be sure to stop the service first before using these commands.

#### Start

Run the following on the command line to start `viam-server`, providing the path to your own configuration file:

```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
sudo viam-server -config /path/to/my/config.json
```

> If you followed our [installation steps](/installation/#install-viam-server), your robot's configuration file is available at <file>/etc/viam.json</file>.

Reviewer: ```suggestion
If you followed these [installation steps](/installation/#install-viam-server), your robot's configuration file is available at <file>/etc/viam.json</file>.
```

the/these - to avoid the first person pronoun.

You can provide this path in the above command, or move the configuration file to a desired location and change the path in this command accordingly.
If you don't yet have a configuration file, you can [build a new configuration file](/appendix/local-configuration-file/).

Note that on a Raspberry Pi, `viam-server` must always run as `root` (using `sudo`) in order to access the DMA subsystem for GPIO.
When running `viam-server` from your home directory on a Linux computer, you do not need to use `sudo`.

#### Stop

> Type **Ctrl + C** on your keyboard within the terminal session where you are running `viam-server` to stop it.

Reviewer: ```suggestion
Press **Ctrl + C** on your keyboard within the terminal session where you are running `viam-server` to stop it.
```

{{% /tab %}}

{{% tab name="macOS"%}}

### Run from the command line

> After [installation](/installation/#install-viam-server), `viam-server` can be run directory on the command line.

Reviewer: ```suggestion
After [installation](/installation/#install-viam-server), `viam-server` can be run directly on the command line.

```

{{% /tab %}}
{{< /tabs >}}

## View `viam-server` logs

`viam-server` writes log messages as it starts up and runs, providing useful information when managing or troubleshooting the process.
Use the following commands to view these log messages locally on your system.

{{< alert title="Tip" color="tip" >}}
If your system is able to connect with the Viam app, logs can also be viewed in the **Logs** tab on [the Viam app](https://app.viam.com/).
{{< /alert >}}

Select the tab below for your platform:

{{< tabs name="View logs">}}
{{% tab name="Linux"%}}

### As a system service

If you are running `viam-server` as a system service, run the following command to view log messages:

```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
sudo journalctl --unit=viam-server
```

Use the arrow keys to page vertically or horizontally through the log messages.

You can also view log messages specific to `viam-server` in the `syslog` with the following command:

```sh {id="terminal-prompt" class="command-line" data-prompt="$"}

> grep viam-server /var/log/syslog

Author: Q: This might only be correct for Debian-based distros. Do we pipe to the local logging facility - in which case this would be `/var/log/messages` on RHEL&friends, for example?

Maybe I should remove this alternate option, here and on line 177-181. LMK.

Reviewer 2: Yeah, and only then if syslog forwarding is enabled in journald's config.

We do no explicit system logging, we rely on systemd to do that. (That'll change with the agent I'm working on>) Other distros without  systemd aren't even supported with our install script.

{{% /tab %}}
{{< /tabs >}}

> ## View `viam-server` logs

Author: Moved troubleshooting content out to Troubleshooting, as it was not manage-specific, but also helps with installation.

sudo journalctl --unit=viam-server
```

Use the arrow keys to page vertically or horizontally through the log messages.

You can also view log messages specific to `viam-server` in the `syslog` with the following command:

```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
grep viam-server /var/log/syslog
```

### From the command line

If you are running `viam-server` on the command line, log messages are written to standard out (`stdout`) in the same terminal session you started `viam-server` in.

You can also view log messages specific to `viam-server` in the `syslog` with the following command:

```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
grep viam-server /var/log/syslog
```

{{% /tab %}}

{{% tab name="macOS" %}}

### From the command line

When running `viam-server` on the command line in macOS, log messages are written to standard out (`stdout`) in the same terminal session you started `viam-server` in.

You can also access the local `viam-server` log file using the following command:

```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
cat $(brew --prefix)/var/log/viam.log

> ```

Reviewer 2: This looks to duplicate the bit below in the "as a system service" section.

{{% /tab %}}
{{< /tabs >}}

## View `viam-server` logs

`viam-server` writes log messages as it starts up and runs, providing useful information when managing or troubleshooting the process.
Use the following commands to view these log messages locally on your system.

{{< alert title="Tip" color="tip" >}}

> If your system is able to connect with the Viam app, logs can also be viewed in the **Logs** tab on [the Viam app](https://app.viam.com/).

Reviewer: ```suggestion
If your system is able to connect with the Viam app, you can also view logs in the **Logs** tab on [the Viam app](https://app.viam.com/).
```

Have you installed vale in vscode? It should be underlining all of these passive voices

Author: Installed but ... was disabled! Enabled now, thank you!
